examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed/dem-packing-in-spouted-cylinder-bed-generator.prm

```diff
@@ -66,7 +66,6 @@ end
 #---------------------------------------------------
 
 subsection lagrangian physical properties
-  set gz                       = -9.81
```

Comment:
Nice catch, I have run a script to change parameters, dunno what happened. I will double check everything then!

Reply:
Oh, my script was looking for gx. But that's super weird that the test has passed.

Reply:
Mhm, that's a thing I can not prevent in the check for the deprecated parameter thing tho because it checks for the gx parameter also. First time a see a prm without all the direction, but it is actually legit. Mhm

Reply:
This is not a unit test, it's an example. So it has passed because essentially the syntax is still correct
Don't worry, we caught it, nothing bad happened.

Reply:
Oh, true

Reply:
Make me thing I need to modify the doc of example also